joplin/templates/wagtailadmin/chooser/browse.html

```diff
@@ -0,0 +1,46 @@
+{% load i18n %}
+{% if page_types_restricted %}
```

Comment:
I'm not sure what page_types_restricted means, I'm thinking it means we set available page types in the model, which makes me curious as to some of the strings in this logic

Reply:
Hmm I'll try it out but that may have just been a typo on my end, but generally yes that is whats its used for (like for setting the help text appropriately below)

Reply:
So I decided to leave this because its a way to avoid checking all these if statements if the picker isn't even restricted in the first place. Probably gives a minor performance benefit but I don't think it hurts either.